### DISCUSSION TITLE
TypeError problem in a new Next Project

### DISCUSSION THREAD

**Discussion by nicolas-CaC** [vercel/next.js#42805] [Help]
Hi, i create my app
W but when i run yarn dev appears this problem:

TypeError: Cannot read properties of null (reading 'length')
    at eval (webpack-internal:///./node_modules/next/dist/client/dev/error-overlay/hot-dev-client.js:262:55)  

Why?

**Comment by meotimdihia** [comment]
I have the same problem: 
```
event - compiled client and server successfully in 375 ms (1460 modules)
warn  - Fast Refresh had to perform a full reload. Read more: https://nextjs.org/docs/basic-features/fast-refresh#how-it-works
TypeError: Cannot read properties of null (reading 'length')
    at eval (webpack-internal:///./node_modules/next/dist/client/dev/error-overlay/hot-dev-client.js:262:55
```

**Comment by karlhorky** [comment]
@balazsorban44 maybe this is a new issue that has happened in 13.0.3?

Also reported on Stack Overflow: https://stackoverflow.com/questions/74415937/nextjs-typeerror-cannot-read-properties-of-null-reading-length

**Reply by karlhorky** [reply]
@nicolas-CaC or @meotimdihia, if this is confirmed as an issue, would you be open to reporting this as an issue over at https://github.com/vercel/next.js/issues ?

**Comment by icyJoseph** [comment]
Does anyone have a repository where this is happening?

**Reply by DanPresa** [reply]
It happends when you create a new project. I don't know if it only happends with typescript

**Comment by jonioni** [comment]
Seems the issue has just been resolved in #43145 and the patch will be released in v13.0.5.

**Comment by Swaraj-Nalabale** [comment]


**Reply by DanPresa** [reply]
That-s not true. I haven't deleted index file and I'm having the issue
